Makefile

```
@@ -14,6 +14,13 @@ install_deps:  ## Install all dependencies.
 setup:  ## Install all dependencies and setup pre-commit
 	make install_deps
 	pre-commit install
+	make .env
+
+.env:  ## Generate .env file
+	@cp .env.example $@
```

This will copy without failing and informing the user that they need to adjust: will that work, i.e, the example .env will result in a test run which works? Otherwise I would add a error message and fail the target:

```suggestion
	@cp .env.example $@
	@echo "PLEASE ADJUST $@ to your liking and rerun!"
	@false
```

---

The test run will raise error because it could not find the dbt profile. Do we think it is an expected behavior ? I would say this is ok.